dev.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:convert';
+import 'dart:io';
+
+void main(List<String> arguments) async {
+  if (arguments == null) {
+    throw ArgumentError('dart dev.dart <proj-name-from-project_config.json>');
```

Comment:
in the console this would look like:
`Invalid argument(s): dart dev.dart <proj-name-from-project_config.json>`
which doesn't read so well because "Invalid argument(s):" creates the expectation of "I'm going to get some info about what's wrong with the arguments, whereas the message is a usage message (the user will likely interpret it as such, but it will be a bit confusing at the beginning. A simple fix would be to just prefix the message with "Usage: "
```suggestion
    throw ArgumentError('Usage: dart dev.dart <proj-name-from-project_config.json>');
```

I'm also wondering if, instead of passing in the project name, and then checking it against some files in a default location, why not pass those files in as arguments to this script? That way, the user can be explicit as to which project configs files they would like to pass in, with the default ones being as an example (which you could give in the readme). What do you think?

Reply:
or you could make the `config/base_config.json` and `config/project_config.json` as the default for running `dev.dart` without any args, and if the user is passing in other config files, it will use those?

Reply:
I thought of having one `base_config.json` and just one `project_config.json` where I will add configs that are project specific than a config file per project. That way I could avoid building logic to identify the config files and at the same time get an overview of all project configs at the same time. Would that be okay? 

I have changed the dev.dart such that if no arguments are provided, it simply takes the first project as the default one under `project_config.json`. Let me know if this is okay.